scripts/deploy.py

```diff
@@ -150,6 +152,17 @@ def preprocess_release():
     with open(os.path.join('assets', 'constants.js'), 'w+') as new_assets_file:
         new_assets_file.write(content)
 
+    # Removes the version field from app.yaml.
+    print 'Removing the version field from app.yaml ...'
+    with open('app.yaml', 'r') as f:
+        content = f.read()
+        assert content.count('version: default') == 1
+    os.remove('app.yaml')
+    content = content.replace('version: default', '')
+    with open('app.yaml', 'w+') as f:
```

Comment:
I think if you just open the file in 'w' mode, it should overwrite everything?

Reply:
Done. Thanks!